specification/ml/_types/TrainedModel.ts

```diff
@@ -131,34 +143,49 @@ export class TrainedModelSizeStats {
 
 export class TrainedModelDeploymentNodesStats {
   /** The average time for each inference call to complete on this node. */
-  average_inference_time_ms: DurationValue<UnitFloatMillis>
+  average_inference_time_ms?: DurationValue<UnitFloatMillis>
+
+  average_inference_time_ms_last_minute?: DurationValue<UnitFloatMillis>
+
+  /** The average time for each inference call to complete on this node, excluding cache */
+  average_inference_time_ms_excluding_cache_hits?: DurationValue<UnitFloatMillis>
+
   /** The number of errors when evaluating the trained model. */
-  error_count: integer
+  error_count?: integer
   /** The total number of inference calls made against this node for this model. */
-  inference_count: integer
+  inference_count?: long
+
+  inference_cache_hit_count?: long
+
+  inference_cache_hit_count_last_minute?: long
+
   /** The epoch time stamp of the last inference call for the model on this node. */
-  last_access: long
+  last_access?: long
```

Comment:
```suggestion
  last_access?: EpochTime<UnitMillis>
```

Might as well be `UnitNanos`. We have to double check.

Reply:
it's indeed nano

Reply:
same for start_time